packages/hydrooj/src/handler/problem.ts

```diff
@@ -80,6 +80,10 @@ registerResolver(
         return pdoc;
     },
 );
+registerResolver('Query', 'problems(ids: [Int])', '[Problem]', async (arg, ctx) => {
+    const res = await problem.getList(ctx.args.domainId, arg.ids, undefined, undefined, undefined, undefined, true);
+    return Object.keys(res).map((id) => res[+id]);
```

Comment:
is permission control required?
e.g. query hidden problem (in contest_edit) for admin user or problem creator